README.md

```diff
@@ -1,11 +1,19 @@
-## System tests
+## What is system-tests?
 
-Workbench designed to run advanced tests (integration, smoke, functional, fuzzing and performance)
+A workbench designed to run advanced tests (integration, smoke, functional, fuzzing and performance) against our suite of dd-trace libraries.
+
+When making changes to dd-trace, you'll commonly need to run the unmerged changes against the system tests (to ensure the feature is up to spec). But because system tests are outside of dd-trace-xx repos, testing against unmerged changes is not so straightforward. Various approaches  to this "chicken or the egg" problem are detailed [here](https://github.com/DataDog/system-tests/blob/main/docs/execute/how-to-approach-changes.md).
```

Comment:
I think that precising that here is too early, `docs/README.md` is the entry point, and should be kept to a very high level POV. If we put here any usage details, it will dilute the important point. WDYT ?

Reply:
makes sense! will reconsider this..